### DISCUSSION TITLE
Karabiner-Elements prevents ⌘` from moving focus to next window

### DISCUSSION THREAD

**Discussion by Caroline-Elisa** [pqrs-org/Karabiner-Elements#3785] [Q&A]
As soon as I quit Karabiner-Elements I can use this combination again. Issue persists even when I have no modifications.

Is there a way to resolve this?

I am on macOS 14 Sonoma with Karabiner-Elements 14.13.0 

Thank you

**Comment by gforien** [comment]
Late reply but the issue probably originates from your configuration
Can you provide it ?
